frontend/public/components/build-config.tsx

```diff
@@ -110,12 +118,56 @@ BuildConfigsDetailsPage.displayName = 'BuildConfigsDetailsPage';
 const tableColumnClasses = [
   '',
   '',
-  'pf-m-hidden pf-m-visible-on-md',
+  'pf-m-hidden pf-m-visible-on-lg',
+  'pf-m-hidden pf-m-visible-on-lg',
+  'pf-m-hidden pf-m-visible-on-lg',
   'pf-m-hidden pf-m-visible-on-lg',
   Kebab.columnClass,
 ];
 
-const BuildConfigsTableRow: React.FC<RowFunctionArgs<K8sResourceKind>> = ({ obj }) => {
+const displayDurationInWords = (start: string, stop: string): string => {
```

Comment:
We have a `getDuration` util in the pipelines-plugin package. Can we use that instead of writing a new one again? We can move that to the console-shared package and use that across the packages.

Reply:
I discussed this with Aneesh already last week. I added this here as well https://github.com/openshift/console/pull/12809/files#diff-947d0c2c07ad6a3b08b3af8f3418515376b26b91fb695ddf1eee9f9aff4c50d5, and I think its a valid comment, but on the other side we can keep it simple and accept this for now, wdyt?

Reply:
I created a follow-up story and added this as additional comment. Would be great to improve this there. For me everything else looks fine, so I will add lgtm and hope that's fine for you as well. :)